scripts/topiary-analyze-cohort

```diff
@@ -54,65 +54,77 @@ from .mutation_report import print_mutation_report
 parser = argparse.ArgumentParser()
 
 input_group = parser.add_mutually_exclusive_group(required=True)
-input_group.add_argument("--variant-input-dir",
-   type=str,
-   help="Directory containing MAF or VCF input files")
+input_group.add_argument(
```

Comment:
For me this is the first time looking at this, worth commenting on what goes in the `input_group` vs general args?